server/models/procedures/stock.sql

```diff
@@ -528,70 +529,80 @@ CREATE PROCEDURE `getCMM` (
 
 
-  SET @sum_consumed_quantity = 0;
-  SET @sum_stock_day = 0;
-  SET @sum_consumption_day =0;
-  SET @sum_stock_out_days = 0;
+  SET _sum_consumed_quantity = 0;
+  SET _sum_stock_day = 0;
+  SET _sum_consumption_day =0;
+  SET _sum_stock_out_days = 0;
 
-  SET @sum_days = DATEDIFF( @end_date,  @start_date) + 1;
-  SET @number_of_month = (DATEDIFF(@end_date,  @start_date) + 1)/30.5;
+  SET _sum_days = DATEDIFF(_end_date,  _start_date) + 1;
+  SET _number_of_month = (DATEDIFF(_end_date,  _start_date) + 1)/30.5;
 
 
   SELECT
-      SUM(cmm_data.out_quantity) AS sum_consumed_quantity,
+    SUM(cmm_data.out_quantity) AS sum_consumed_quantity,
     SUM(IF(cmm_data.out_quantity <> 0, cmm_data.frequency, 0)) AS sum_consumption_day,
     SUM(IF(cmm_data.in_stock_quantity = 0, cmm_data.frequency, 0)) AS sum_stock_out_days,
     SUM(IF(cmm_data.in_stock_quantity <> 0, cmm_data.frequency, 0)) AS sum_stock_day
-  INTO @sum_consumed_quantity, @sum_consumption_day, @sum_stock_day, @sum_stock_out_days
+  INTO _sum_consumed_quantity, _sum_consumption_day, _sum_stock_day, _sum_stock_out_days
   FROM (
       SELECT x.start_date, x.end_date, x.in_quantity, x.out_quantity, x.in_stock_quantity,
           x.inventory, x.depot, (DATEDIFF(x.end_date, x.start_date) +1) AS frequency
       FROM (
           SELECT
 
-          IF(m.start_date <@start_date, @start_date, m.start_date) AS start_date ,
-          IF(m.end_date > @end_date, @end_date,  IF(m.end_date = @last_inventory_mvt_date AND
-                  @last_inventory_mvt_date < @end_date, @end_date, m.end_date )) AS end_date,
-
+          IF(m.start_date < _start_date, _start_date, m.start_date) AS start_date ,
+          IF(m.end_date > _end_date, _end_date,  IF(m.end_date = _last_inventory_mvt_date AND
+                  _last_inventory_mvt_date < _end_date, _end_date, m.end_date )) AS end_date,
           in_quantity, out_quantity, m.quantity AS `in_stock_quantity`,
           i.text as 'inventory', d.text AS `depot`
           FROM stock_movement_status m
           JOIN depot d ON d.uuid = m.depot_uuid
           JOIN inventory i ON i.uuid = m.inventory_uuid
 
           WHERE i.uuid = _inventory_uuid AND m.depot_uuid = _depot_uuid
-              AND  ((m.start_date >= @start_date) OR  (m.start_date <= @start_date AND m.end_date >= @start_date))
-              AND ( (m.end_date <=  @end_date) OR ( m.start_date < @end_date ))
+             AND  (
+              ((m.start_date >= _start_date) AND  (m.end_date <= _end_date)) OR
+              ((m.start_date >= _start_date) AND  (m.end_date >= _end_date)) OR
+              ((m.start_date <= _start_date) AND  (m.end_date <= _end_date))
+					  )
           ORDER BY d.text
       ) AS `x`
   ) AS `cmm_data`;
 
 
 
-  SET @algo1 = ( @sum_consumed_quantity/(IF((@sum_stock_day = NULL) OR (@sum_stock_day = 0),1, @sum_stock_day)))*30.5;
+  SET @algo1 = ( _sum_consumed_quantity/(IF((_sum_stock_day = NULL) OR (_sum_stock_day = 0),1, _sum_stock_day)))*30.5;
```

Comment:
Could you also replace these `@` variables with `_` variables?

Reply:
Not off hand. I declared them like variables before remplacing ```@``` by ```_```.